### ISSUE TITLE
Provide a way to wipe drives

### ISSUE THREAD

**Issue by Un1q32** [fnuecke/oc2#238]
**Is your feature request related to a problem? Please describe.**
If I break my linux install I want to be able to restore it to its default state instead of having to get a new drive.
**Describe the solution you'd like**
I think when a hard drive with linux already on it is placed in a grid with a scrench it should revert it back to a normal hard drive with no data.

**Comment by atirutw** [comment]
But then where would you install it from?

**Comment by Un1q32** [comment]
@atirut-w placing a drive without linux on it would make it a linux drive, but if it already had linux on it it would be wiped.

**Comment by atirutw** [comment]
Determining if a disk have Linux might be tricky

**Comment by Un1q32** [comment]
> Determining if a disk have Linux might be tricky

there is litterally a seperate item for disks with linux

**Comment by atirutw** [comment]
Yes but once you use them they becomes different. Maybe there are new files, dotfiles, etc. IIRC, these disks have a sort of template filesystem that becomes normal disk with a copy of those templates, sort of like burning an ISO file onto a disk.

**Comment by Un1q32** [comment]
@atirut-w yeah ik.  Using a scrench would destroy everything on it.  It already does it just only works one way.

**Comment by Void-Seeker** [comment]
Keep fresh Sedna Linux drive as a backup, and if you mess up your working drive, boot from backup drive and do `dd if=/dev/vda of=/dev/vdb`

**Comment by Un1q32** [comment]
> Keep fresh Sedna Linux drive as a backup, and if you mess up your working drive, boot from backup drive and do `dd if=/dev/vda of=/dev/vdb`

That works but I shouldn't have to keep a backup drive all the time it should be built into the mod.

**Comment by Void-Seeker** [comment]
I wish I could install Linux on an empty hard drive with a wrench IRL...

**Comment by fnuecke** [comment]
Linux disk (and generally any crafted/preconfigured media) can now be crafted with the wrench to turn it back into a regular one.
